backend/capellacollab/projects/crud.py

```diff
@@ -42,6 +43,11 @@ def create_project(
     db: Session, name: str, description: str | None = None
 ) -> DatabaseProject:
     slug = slugify(name)
+    if (
+        db.query(DatabaseProject).filter(DatabaseProject.slug == slug).first()
+        is not None
+    ):
+        raise HTTPException(409, "Slug already used.")
```

Comment:
Please try to avoid the `HTTPException` without a detail body. With this variant, we're not able to display the error in the frontend (instead, a unknown error appears, which is not really user friendly). You can have a look at https://github.com/DSD-DBS/capella-collab-manager/pull/227 to see how to use the `detail` parameter of the `HTTPException`.